packages/contracts-bedrock/contracts/L2/L2StandardBridge.sol

```diff
@@ -273,4 +276,49 @@ contract L2StandardBridge is StandardBridge, Semver {
         emit DepositFinalized(_remoteToken, _localToken, _from, _to, _amount, _extraData);
         super._emitERC20BridgeFinalized(_localToken, _remoteToken, _from, _to, _amount, _extraData);
     }
+
+    function _emitBITBridgeInitiated(
+        address _localToken,
+        address _remoteToken,
```

Comment:
The parameters (`_localToken` and `_remoteToken`) seem useless here.

Reply:
This function is a override function which is from StandardBridge.And L1Bridge also need to use the function